doc/releases/migration-guide-4.0.rst

```diff
@@ -41,6 +41,11 @@ Mbed TLS
 * The Kconfig options ``CONFIG_MBEDTLS_TLS_VERSION_1_0`` and ``CONFIG_MBEDTLS_TLS_VERSION_1_1``
   have been removed because Mbed TLS doesn't support TLS 1.0 and 1.1 anymore since v3.0. (:github:`76833`)
 
+* The Kconfig option ``CONFIG_MBEDTLS_SSL_EXPORT_KEYS`` was removed because the
+  corresponding build symbol was removed from Mbed TLS since revision 3.1.0.
+  ``MBEDTLS_SSL_EXPORT_KEYS`` is assumed to be default on in Mbed TLS.
```

Comment:
I think this could be rephrased to be more clear. To me this sounds like the macro may still exist in Mbed TLS, which it doesn't. Maybe something like:
```suggestion
  Mbed TLS has the functionality that ``MBEDTLS_SSL_EXPORT_KEYS`` used to enable enabled by default, so uses of ``CONFIG_MBEDTLS_SSL_EXPORT_KEYS`` can simply be removed.
```

Reply:
Mmmm, I still prefer my version: it has the same meaning and less "enable" repetitions. But if you strongly suggest to change this, I can of course

Reply:
It's fine. I just find the `assumed to be default on` possibly a bit misleading. Maybe just changing it to `assumed to be enabled` to at least remove the `default` would be more clear and okay for you?